src/app.rs

```diff
@@ -430,26 +430,25 @@ impl App {
                 [OutputComponent::Numbers].iter().cloned().collect()
             } else if matches.is_present("plain") {
                 [OutputComponent::Plain].iter().cloned().collect()
-            } else if matches.is_present("style") {
-                values_t!(matches.values_of("style"), OutputComponent)?
+            } else {
+                let env_style_components: Option<Vec<OutputComponent>> =
+                    transpose(env::var("BAT_STYLE").ok().map(|style_str| {
+                        style_str
+                            .split(",")
+                            .map(|x| OutputComponent::from_str(&x))
+                            .collect::<Result<Vec<OutputComponent>>>()
+                    }))?;
+
+                values_t!(matches.values_of("style"), OutputComponent)
+                    .ok()
+                    .or(env_style_components)
```

Comment:
Yeah you know I really should have done that from the beginning... my bad

Reply:
No worries! Thank you very much for your contribution. I took the liberty to change things myself. Hope that's fine with you!

Reply:
I mean I just try to help out because I like using bat and your change was clearly for the better, it's all good. I learn something new every time as well so thank you!